### PR TITLE
Feat/SP-931 Add CycloneDX export

### PR DESCRIPTION
**Pull request by isasmendiagus**
* Adds the option to export a CycloneDX SBOM

### CODE REVIEW

File: src/main/task/export/Export.ts [hunk 1]
```diff
@@ -13,6 +13,9 @@ import { ITask } from '../Task';
 import { IExportResult } from '../../modules/export/IExportResult';
 import { ExportFormat, InventoryType } from '../../../api/types';
 import { Crypto } from '../../modules/export/format/Crypto';
+import { CycloneDX } from '../../modules/export/format/CycloneDX';
+import { workspace } from '../../workspace/Workspace';
```
Comment (agustingroh) [inline src/main/task/export/Export.ts:17]: Should we add the ExportModel and project in all the formats in the future?

**Review by agustingroh — CHANGES_REQUESTED** [review]
I tried to export the CycloneDX from the detected and identified section but the UI is throwing the following error.
I made the following steps before test the branch:
- npm  install --legacy-peer-deps
- npm run




image

image

**Comment by isasmendiagus** [comment]
> I tried to export the CycloneDX from the detected and identified section but the UI is throwing the following error. I made the following steps before test the branch:
> 
>     * npm  install --legacy-peer-deps
> 
>     * npm run
> 
> 
> image
> 
> image

There was a silly bug. The license declared when you start a new project it's optional, that caused the crash

**Comment by isasmendiagus** [comment]
> I tried to export the CycloneDX from the detected and identified section but the UI is throwing the following error. I made the following steps before test the branch:
> 
>     * npm  install --legacy-peer-deps
> 
>     * npm run
> 
> 
> image
> 
> image

Fix in last commit, please check it again @agustingroh

**Review by agustingroh — APPROVED** [review]

**Pull request merged**